### PR TITLE
Regen bindings with new gir {g,s}etter annotations

### PR DESCRIPTION
**Pull request by A6GibKm**

### CODE REVIEW

File: gdk4/src/auto/display.rs [hunk 1]
```diff
@@ -290,11 +295,6 @@ pub trait DisplayExt: IsA<Display> + sealed::Sealed + 'static {
         }
     }
 
-    #[doc(alias = "input-shapes")]
-    fn is_input_shapes(&self) -> bool {
```
Comment (sdroege) [inline gdk4/src/auto/display.rs:294]: This is gone now and merged with the getter function. Nice!

**Review by sdroege — COMMENTED** [review]
Looks nice to me, some redundant properties got removed (so you probably don't want to backport this @bilelmoussaoui )

**Comment by bilelmoussaoui** [comment]
> Looks nice to me, some redundant properties got removed (so you probably don't want to backport this @bilelmoussaoui )

yeah, not planning to do any release till the one in mid-july anyways. Will review and merge things once I am done with dealing with the ffi renames in gstreamer

File: gtk4/Gir.toml [hunk 2]
```diff
@@ -1426,6 +1426,15 @@ status = "generate"
...
         nullable = false
+    [[object.function]]
+    name = "set_from_paintable"
+    rename = "set_paintable"
+    [[object.function]]
+    name = "set_from_icon_name"
+    rename = "set_icon_name"
+    [[object.function]]
+    name = "set_from_resource"
+    rename = "set_resource"
```
Comment (bilelmoussaoui) [inline gtk4/Gir.toml:1437]: The changes here looks good to me, although they should be in a separate commit + please add an inline comment to explain the reason behind the renaming.

Comment (A6GibKm) [reply gtk4/Gir.toml:1440]: They are already in a second commit, but I expanded the commit message a bit and added a comment.

**Comment by bilelmoussaoui** [comment]
Could you update gir & regen?

**Comment by A6GibKm** [comment]
> Could you update gir & regen?

Done.

File: gtk4/src/auto/drop_down.rs [hunk 3]
```diff
@@ -65,6 +65,8 @@ impl DropDown {
 
     #[doc(alias = "gtk_drop_down_get_enable_search")]
     #[doc(alias = "get_enable_search")]
+    #[doc(alias = "enable-search")]
```
Comment (bilelmoussaoui) [inline gtk4/src/auto/drop_down.rs:68]: Duplicated doc aliases... looks like you have to fix gir :P

Comment (A6GibKm) [reply gtk4/src/auto/drop_down.rs:68]: https://github.com/gtk-rs/gir/pull/1572

Comment (bilelmoussaoui) [reply gtk4/src/auto/drop_down.rs:68]: You can regen now

Comment (A6GibKm) [reply gtk4/src/auto/drop_down.rs:68]: Regenerated.

**Comment by bilelmoussaoui** [comment]
the typos ci is unrelated, will fix in a separate commit. Thanks!
